test/feed-parser.test.js

```diff
@@ -25,3 +25,38 @@ test('Passing an invalid RSS feed URI should error', async () => {
 test('Passing a valid URI, but not a feed URI should error', async () => {
   await expect(feedParser('https://google.ca')).rejects.toThrow('Not a feed');
 });
+
+const assertValidFeed = (feed) => {
+  expect(Array.isArray(feed)).toBeTruthy();
+  expect(feed.length > 0).toBeTruthy();
+};
+
+
+test('Non existant feed failure case.', async () => {
+  try {
+    await feedParser('http://doesnotexists___.com');
+  } catch (err) {
+    expect(err.message).toBe('getaddrinfo ENOTFOUND doesnotexists___.com');
```

Comment:
It's risky to depend on the error message like this.  They can change between versions of the platform, different OSes can have slightly different error messages, error strings can be localized, etc.

Instead, use [`err.code`](https://nodejs.org/api/errors.html#errors_error_code) and one of the [official error codes](https://nodejs.org/api/errors.html#nodejs-error-codes)

Reply:
Thanks sir for your preview,I will be changing it right now.

Reply:
I have made the changes.

Thanks